README.md

```diff
@@ -11,7 +11,10 @@
 [![Documentation Status][rtd]](https://open-sir.readthedocs.io/en/latest)
 
 
-Open-SIR is an Open Source Python project for modelling pandemics and infectious diseases using Compartmental Models, such as the widely used [Susceptible-Infected-Removed (SIR) model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics). 
+Open-SIR is an Open Source Python project for modelling pandemics and
+infectious diseases using Compartmental Models, such as the widely used
+[Susceptible-Infected-Removed (SIR)
+model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics).
```

Comment:
Maybe you can add this link to the top of the file, like these lines:
https://github.com/open-sir/open-sir/pull/60/files#diff-04c6e90faac2675aa89e2176d2eec7d8R4

The markdown would look better imo